docs/auto-enable.md

```diff
@@ -0,0 +1,37 @@
+# Automatically create `pg_stat_monitor` for every newly created database
+
+In PostgreSQL, a database is created by copying the [system template database](https://www.postgresql.org/docs/current/manage-ag-templatedbs.html) `template1`. To automatically create the `pg_stat_monitor` extension for every newly created database, you can create a new system template database, create the extension for it and define this template when you create the database.  
+
+Your user must have the superuser privileges.
+
+Here's how to do it:
+
+1. Create a database to serve as the template.
+
+    ```{.bash data-prompt="$"}
+    $ createdb my-template
+    ```
+
+2. Connect to this database and create the `pg_stat_monitor` extension
+
+    ```{.bash data-prompt="$"}
+    $ psql -d my-template -c 'create extension pg_stat_monitor;'
```

Comment:
This is just stylistic, but this example uses the `psql` command, while later examples only show SQL statements

Reply:
What would you suggest: stick to psql or advise to connect to psql and use SQL statements going forward?

Reply:
Looking at our existing documentation, it usually uses SQL statements, and doesn't even mention the "connect to the database using psql..." part. So I would follow that style. Our documentation site also has some syntax highlighting for it, which is another plus. And similarly, it's a good idea to keep sql keywords upcase (for example WHERE, CREATE EXTENSION, and so on).